src/DETHRACE/constants.h

```diff
@@ -351,6 +351,6 @@ typedef enum keymapcodes {
 
 #define SHADOW_MAX_RENDER_DISTANCE 36.0f
 
-#define DOUBLESIDED_FLAG_COLOR_MAP (br_pixelmap*)12345
+#define DOUBLESIDED_USER_FLAG (void*)12345
```

Comment:
Perhaps we need to do something as following?
```c
typedef struct {
    int flags;
} dethrace_material_props;

#define DR_MATUFLAGS_DOUBLE_SIDED 0x01
```

A nuisance with this approach is the need for allocation, but should be do-able when using `BrResAllocate` (which will automatically de-allocate when the parent is de-allocated).

Reply:
your idea is better in all ways :) 

However, since `(void*)12345` is what the OG code does, I'd leave refactoring to be cleaner to a different repo/branch

Reply:
Oh, then it's totally fine. I thought this was custom code to fix the lanterns.

Reply:
ah cool. Yeah, I'm holding firm to any code in `src/DETHRACE` that doesn't come from OG has to be wrapped in the FIX_BUGS macro